README.md

```diff
@@ -90,11 +91,18 @@ controller.add(&hisThread);
 controller.add(&sensorReadings);
 ...
 ```
+or
+```c++
+// Instantiate a new StaticThreadController
+StaticThreadController<2> controller (Thread(my_callback, my_interval), Thread(his_callback, his_interval));
```

Comment:
Put one example of an inline instantiation, and some stored thread. Like:
`StaticThreadController<2> controller (Thread(callback, 100), aCreatedThread);`

It will clarify cases of pre-created threads